ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -51,6 +51,8 @@ let prototyped_of_field = function
       Some "22.26.0"
   | "VTPM", "persistence_backend" ->
       Some "22.26.0"
+  | "SM", "host_pending_features" ->
+      Some "24.36.0-next"
```

Comment:
Does this need to be 24.37.0?

Reply:
This will be changed to 24.37.0 whenever we tag and release 24.37.0